src/server/common/model/answer/email-address.test.js

```diff
@@ -0,0 +1,113 @@
+import { EmailAddress } from './email-address.js'
+
+const validEmailAddressPayload = {
+  emailAddress: 'test@somewhere.com'
+}
+
+const validEmailAddresses = [
+  'email@domain.com',
+  'email@domain.COM',
+  'firstname.lastname@domain.com',
+  "firstname.o'lastname@domain.com",
+  'email@subdomain.domain.com',
+  'firstname+lastname@domain.com',
+  '1234567890@domain.com',
+  'email@domain-one.com',
+  'email-with-dash@domain-one.com',
+  '_______@domain.com',
+  'email@domain.name',
+  'email@domain.superlongtld',
+  'email@domain.co.jp',
+  'firstname-lastname@domain.com',
+  'info@german-financial-services.reallylongarbitrarytldthatiswaytoohugejustincase',
+  'technically..valid@domain.com',
+  'info@german-financial-services.vermögensberatung',
+  'japanese-info@例え.テスト'
+]
+const invalidEmailAddresses = [
+  'plainaddress',
+  '@no-local-part.com',
+  'no-at.domain.com'
+]
+
+describe('EmailAddress', () => {
+  describe('validate', () => {
+    it('should return true for valid email address', () => {
```

Comment:
move the `it` into the for - so it runs a seperate test for each email (this is consistent with how we did it for postcodes)

Reply:
done